src/traces/scatter/attributes.js

```diff
@@ -152,6 +156,16 @@ module.exports = {
                 'Sets the style of the lines. Set to a dash string type',
                 'or a dash length in px.'
             ].join(' ')
+        },
+        simplify: {
+            valType: 'boolean',
+            dflt: true,
+            role: 'info',
+            description: [
+                'Simplifies lines by removing nearly-collinear points. When transitioning',
+                'lines, it may be desirable to disable this so that the number of points',
+                'along the resulting SVG path is unaffected.'
+            ].join(' ')
```

Comment:
Can you add a quick test for the `simplify: false` case in the scatter [suite](https://github.com/plotly/plotly.js/blob/c47c67ebf99d0608e52ec12f7bb46338145999d2/test/jasmine/tests/scatter_test.js#L198)? No need for anything fancy.